src/scripts/experience/explore-mode.js

```diff
@@ -286,13 +286,24 @@ define(['pixi', 'experience/mode', 'experience/issue'], function (PIXI, Mode, Is
         }
     };
 
+    ExploreMode.prototype._focusIssue = function(issue) {
```

Comment:
So the "experience" view refers to the big canvas with all the issue dots on them. At any given point, the experience has a specific "mode" activated. ExploreMode is the mode with the large cloud of dots floating around.

Each mode runs `_onStartShow` when it's displayed, and `_onStartHide` when it's being hidden as another mode is activated.

Generally the methods on a mode are either initialization/cleanup of the mode, or signal handlers for reacting to user input, such as scrolling or clicking.